Tests/TSCBasicTests/ProcessTests.swift

```diff
@@ -255,7 +264,7 @@ class ProcessTests: XCTestCase {
 
     func testStdin() throws {
         var stdout = [UInt8]()
-        let process = Process(args: script("in-to-out"), outputRedirection: .stream(stdout: { stdoutBytes in
+        let process = Process(args: script("in-to-out"), environment: env(), outputRedirection: .stream(stdout: { stdoutBytes in
```

Comment:
is there a way to make this the default when not otherwise specified?

Reply:
I guess we could create an extension for the purpose of the tests?